static/app/components/events/eventReplay/replayInlineOnboardingPanel.tsx

```diff
@@ -0,0 +1,124 @@
+import {useState} from 'react';
+import styled from '@emotion/styled';
+
+import replaysInlineOnboarding from 'sentry-images/spot/replays-inline-onboarding.svg';
+
+import Button from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import localStorage from 'sentry/utils/localStorage';
+import {useReplayOnboardingSidebarPanel} from 'sentry/utils/replays/hooks/useReplayOnboarding';
+
+const LOCAL_STORAGE_KEY = 'replay-preview-onboarding-hide-until';
+const SNOOZE_TIME = 1000 * 60 * 60 * 24 * 7; // 1 week
+const DISMISS_TIME = 1000 * 60 * 60 * 24 * 365; // 1 year
+
+function getHideUntilTime() {
+  const val = localStorage.getItem(LOCAL_STORAGE_KEY);
+  const time = val ? new Date(val || '').getTime() : 0;
+  return time || 0; // Time could be `NaN`
+}
+
+function setHideUntilTime(offset: number) {
+  localStorage.setItem(LOCAL_STORAGE_KEY, new Date(Date.now() + offset).toISOString());
```

Comment:
Would it be better to save this as unix timestamp?

Reply:
yeah, that's easier.